vms/proposervm/block.go

```diff
@@ -275,6 +275,7 @@ func (p *postForkCommonComponents) buildChild(
 		zap.Stringer("blkID", child.ID()),
 		zap.Stringer("innerBlkID", innerBlock.ID()),
 		zap.Uint64("height", child.Height()),
+		zap.Uint64("pChainHeight", pChainHeight),
```

Comment:
This was useful for debugging... I think it makes sense for it to be logged normally.